src/actions/google/drive/sheets/google_sheets.ts

```diff
@@ -173,6 +173,10 @@ export class GoogleSheetsAction extends GoogleDriveAction {
                             reject(e)
                         })
                     }
+                    const lineData = line.map((v: any) => {
+                        v = (v as string).replace(/\"/g, "\"\"")
```

Comment:
This might be relevant. tl;dr - maybe use tr() instead?
https://blog.appsignal.com/2019/08/20/clean-up-strings.html

Reply:
Nevermind, for JS this looks to be as fast as we can get...
https://stackoverflow.com/questions/9750338/dynamic-vs-inline-regexp-performance-in-javascript